test/examples/ucp_client_server.c

```diff
@@ -109,7 +110,7 @@ void set_listen_addr(struct sockaddr_in *listen_addr)
     memset(listen_addr, 0, sizeof(struct sockaddr_in));
     listen_addr->sin_family      = AF_INET;
     listen_addr->sin_addr.s_addr = INADDR_ANY;
-    listen_addr->sin_port        = server_port;
+    listen_addr->sin_port        = htons(server_port);
```

Comment:
looks like this has already been a part of upstream - https://github.com/openucx/ucx/pull/3533

Reply:
probably git merged this away.. anyway we can remove from this PR